spec/garbage.dd

```diff
@@ -447,27 +447,27 @@ $(H2 $(LNAME2 gc_config, Configuring the Garbage Collector))
 
 $(H2 $(LNAME2 precise_gc, Precise Heap Scanning))
 
-    $(P If you select `precise` as the garbage collector via the options above, type
+    $(P Selecting `precise` as the garbage collector via the options above means type
         information will be used to identify actual or possible pointers or
         references within heap allocated data objects. Non-pointer data will not
         be interpreted as a reference to other memory as a "false pointer". The collector
         has to make pessimistic assumptions if a memory slot can contain both a pointer or
         an integer value, it will still be scanned (e.g. in a `union`).
     )
 
-    $(P If you use the GC memory functions from `core.memory`, and plan to use it
-        for data with a mixture of pointers and non-pointer data you should pass the
-        TypeInfo of your allocated struct, class or type as the optional parameter.
+    $(P To use the GC memory functions from `core.memory`
+        for data with a mixture of pointers and non-pointer data, pass the
+        TypeInfo of the allocated struct, class or type as the optional parameter.
```

Comment:
IMO, we really should be using the Oxford comma everywhere. I'd put a comma after "class" in "class or type". We probably aren't using it everywhere, but I think we should insert it when we see it's missing.